packages/codefixes/src/typescript-codefix-collection.ts

```diff
@@ -202,6 +220,10 @@ export class TypescriptCodeFixCollection implements CodeFixCollection {
         continue;
       }
 
+      if (filter.safeFixes) {
+        fix = this.makeCodeFixSafe(fix, diagnostic);
```

Comment:
would it better for this function to return a boolean, and `continue` on `false`, rather than modifying the `changes` property and doing a check for empty changes?

Reply:
It's more flexible to modify changes (we do same in makeCodeFixStrict).

In the next step it expected to return modified changes with 2 items: remove comment + add type somewhere in the file.

Plus codefixe can have few changes and only one can be modified / removed in case we will be using this function for something else in future.

Reply:
in that case, should the 

```
// Covers cases with broken type signatures, like: `() =>`
        const brokenTypeSignatures = /\(\) =>\s*$/i;
        if (brokenTypeSignatures.test(textChanges.newText)) {
          continue;
 }
```

code from strict move here?

Reply:
Type signature provided by codefix isn't broken `type HeadCount = { count: number; };`

But, file became broken after it's added to the file (codefix applied) at the wrong place.

Reply:
Also, we do check for broken signatures / unresolvable types in `makeCodeStrict`

Reply:
Also, we do check for broken signatures / unresolvable types in `annotateWithStrictTypeFromJSDoc` since JSDocs are the source of broken types :)